cookbooks/aws-parallelcluster-slurm/test/controls/config_slurmd_systemd_service_spec.rb

```diff
@@ -11,7 +11,7 @@
 
 control 'tag:config_systemd_slurmd_service' do
   title 'Check the basic configuration of the systemd slurmd service'
-  only_if { !os_properties.on_docker? }
+  only_if { !os_properties.on_docker? && instance.compute_node? }
```

Comment:
Nitpicks:
1. Should we also change the control name to reflect it's only for compute nodes?
2. I find it more readable using multiple `only_if` statements. If you agree, could you try and confirm it actually works?